src/config/config.ts

```diff
@@ -2,19 +2,17 @@ import { ethers } from "ethers";
 
 import MemoryQueue from "../batch/memoryQueue";
 import { QueueInterface } from "../batch/queue";
-import MemoryStore from "../storage/memoryStorage";
 import { StorageInterface } from "../storage/storage";
 
 export interface Config {
   provider?: ethers.providers.Provider;
   signer?: ethers.Signer;
   queue: QueueInterface;
-  store: StorageInterface;
+  store?: StorageInterface;
```

Comment:
Is this meant to be optional?  It throws in get & put if there's no store in the config.

Reply:
Yeah I was going to do that in the next PR. I wanted to keep this as simple as possible.

Reply:
oh I see ok all that makes sense, thanks.